packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,257 @@
+    >
+      {selectedItemName}
+    </Tag>
+  )}
+</Box>;
+```
+
+### With Different Label Positions
+
+While things are flawless when you give visible label (highly recommended), If there are scenarios where you are not able to give label to inputs (questionable but ok), you can use `labelPosition: 'inside-input'` prop.
+
+| **Code**                                                       | **Description**       | **Single Select**                                                                                                | **Multi Select**                                                                                                 |
+| -------------------------------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete label="Status" /> `                             | With Label Present    | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/0c559daa-5cdc-4d71-9221-6073b7969b1b" /> | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/9314fd11-96b7-46ef-963f-283b393cb9fd" /> |
+| `<AutoComplete label="Status" labelPosition="inside-input" />` | Without Label Present | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/2cb1d659-4a98-4f63-96d8-204ead4a681f" /> | <img width="500" src="https://github.com/razorpay/blade/assets/30949385/0b219c4c-2df2-4cfe-a32d-4c02809ab440" /> |
+
+### With Different Inactive - Active States
+
+> **Note**
+>
+> Number of items shown in inactive state is auto-calculated based on height and width of the input.
+
+| Code                                    | Inactive                                                                                                                   | Active                                                                                                                     |
+| --------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| `<AutoComplete maxRows="single" />`     | <img width="266" alt="image" src="https://github.com/razorpay/blade/assets/30949385/74b849f3-f1b9-4922-b717-70ffc7ffd56e"> | <img width="276" alt="image" src="https://github.com/razorpay/blade/assets/30949385/055bc4f4-fb57-4129-b27a-0f1966c8ad26"> |
+| `<AutoComplete maxRows="multiple" />`   | <img width="420" alt="image" src="https://github.com/razorpay/blade/assets/30949385/ed71318c-889e-407e-86bf-fe4f5902dedf"> | <img width="416" alt="image" src="https://github.com/razorpay/blade/assets/30949385/8a636a3c-2afe-43a2-b283-c5ef40c14362"> |
```

Comment:
With `maxRows="multiple"` are we limiting the number of rows to 3? If yes, this should be mentioned in the API decisions. If not, how are we deciding where to place the "+x more" indicator?

Reply:
> With maxRows="multiple" are we limiting the number of rows to 3?

yes. Will add it in doc.

Reply:
added a description column in this table with explaination